tests/test_scripts.py

```diff
@@ -106,12 +106,16 @@ def test_merge_contributors(self):
                 identifier_id = oclc_id.identifier,
                 title = "King Kong Ain't Got Nothin On Me"
             )
-            edition.contributors[0].name = "Denzel Washington"
+            [contributor] = list(edition.contributors)
+            contributor.name = "Denzel Washington"
             self.edition1.primary_identifier.equivalent_to(
                 self.script.input_data_source, oclc_id, 1
             )
             self._db.commit()
         eq_(0, len(self.edition1.contributors))
         self.script.merge_contributors(self.edition1.primary_identifier)
         eq_(2, len(self.edition1.contributors))
-        eq_("Denzel Washington", self.edition1.contributors[0].name)
+        eq_(
+            ["Denzel Washington"] * 2, 
+            [x.name for x in self.edition1.contributors]
+        )
```

Comment:
I don't understand why `merge_contributors` creates two distinct `Contributor` objects for Denzel Washington. I'm pretty sure that was the old behavior, and maybe it wasn't as noticeable back when `.contributors` was expected to return the same `Contributor` more than once, but it seems worrying.